src/badge.jsx

```diff
@@ -1,9 +1,54 @@
 import React from 'react';
 import Typography from './styles/typography';
 import getMuiTheme from './styles/getMuiTheme';
-import StylePropable from './mixins/style-propable';
 
-// Badge
+function getStyles(props, state) {
+  const {
+    primary,
+    secondary,
+  } = props;
+
+  const {
+    badge,
+  } = state.muiTheme;
+
+  const [
```

Comment:
You really like ternary monsters :laughing: :laughing: what's wrong with `let` + `if/else` chain? :laughing: